transport/stream.go

```diff
@@ -70,13 +69,14 @@ func WrapConn(c StreamConn, r io.Reader, w io.Writer) StreamConn {
 	return &duplexConnAdaptor{StreamConn: conn, r: r, w: w}
 }
 
-// StreamEndpoint represents an endpoint that can be used to established stream connections (like TCP) to a fixed destination.
+// StreamEndpoint represents an endpoint that can be used to establish stream connections (like TCP) to a fixed
```

Comment:
[StreamEndpoint]

Reply:
Type names should not be wrapped in `[]` in the documentation that describes the type itself.